distutils/dist.py

```diff
@@ -26,6 +25,13 @@
 from .fancy_getopt import FancyGetopt, translate_longopt
 from .util import check_environ, rfc822_escape, strtobool
 
+try:
+    from packaging.utils import canonicalize_name, canonicalize_version
+except ImportError as ex:  # pragma: no cover
+    raise ImportError(
+        "Distutils should be distributed alongside setuptools and its dependencies"
+    ) from ex
```

Comment:
This exception is not really necessary, just adds a bit more verbosity to help debugging if necessary. We might as well just remove the `try...except` wrapper and run the import statement directly.

Reply:
Agreed. Let's just import it naturally.

Reply:
Done in 9d05e00f4.